airbyte-cdk/Dockerfile.connectors

```diff
@@ -0,0 +1,5 @@
+FROM python:3.9-slim
+
+WORKDIR /home/airbyte-cdk
+COPY . ./
```

Comment:
Why not copy `python` instead of `.`? Is there value to copy the license file and the `build` folder?

Reply:
Nope I think we're good to just copy over `python`. Will make that change.